app/javascript/packs/hello_react.jsx

```diff
@@ -0,0 +1,27 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```

Comment:
Could probably stand to delete this file, no?

Reply:
Totally, held onto it to look back on as a starter example